mkdocs/pages/en/devbook/accounts/create-from-mnemonic.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Create from Mnemonic
+---
+
+# Creating Accounts from Mnemonics
+
+This tutorial shows how to create <accounts:> for the Symbol blockchain using a <mnemonic phrase:>.
+
+This approach is commonly used by <HD wallets:> to manage multiple accounts from a single seed.
+
+## Prerequisites
+
+If you have not done so already, start with the [Hello World](../start/hello-world.md) tutorial to make sure your
+development environment is set up correctly.
+
+## Full Code
+
+{% import 'tutorial.jinja2' as tutorial with context %}
+
+{{ tutorial.code_full('devbook/accounts/create-from-mnemonic', ['py', 'js']) }}
+
+## Code Explanation
+
+### Initializing the Facade
+
+{{ tutorial.code_snippet(['py:5:6', 'js:5:6']) }}
+
+The <dy:SymbolFacade> provides access to Symbol's cryptographic operations and network utilities.
+It is initialized with a network name (`testnet` or `mainnet`) to ensure that network-specific values,
+such as <address:>, are generated correctly.
+
+### Defining a Mnemonic
+
+{{ tutorial.code_snippet(['py:8:16', 'js:8:17']) }}
+
+The example checks for an existing mnemonic in the `MNEMONIC` environment variable.
+If the variable is set, the mnemonic is loaded from it.
+Otherwise, a new random mnemonic is generated using <dy:Bip32.random>.
+
+Symbol uses the [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) standard, which represents
+mnemonics as 24 English words selected from a standardized word list.
+These words encode entropy that serves as the foundation for deriving private keys.
+
+!!! warning "Store your mnemonic securely"
+    The mnemonic can be used to regenerate all derived accounts and private keys.
```

Comment:
```suggestion
!!! warning "Store your mnemonic phrase securely"
    The mnemonic phrase can be used to regenerate all derived accounts and private keys.
```

We can informally use "mnemonic" throughout the tutorial, but for serious matters like this warning box, let's use the full name "mnemonic phrase".